examples/014_cyclic_symmetry_example.py

```diff
@@ -0,0 +1,160 @@
+"""
+
+# %%
+# Set up analysis
+# ~~~~~~~~~~~~~~~
+# Setting up the analysis consists of loading the required modules, connecting to the
+# DPF server, and retrieving the example files.
+#
+# Load Ansys libraries and helper functions
+import ansys.dpf.core as dpf
+
+from ansys.dpf.composites.composite_model import CompositeModel
+from ansys.dpf.composites.constants import FailureOutput, Sym3x3TensorComponent
+from ansys.dpf.composites.example_helper import get_continuous_fiber_example_files
+from ansys.dpf.composites.failure_criteria import CombinedFailureCriterion, MaxStressCriterion
+from ansys.dpf.composites.layup_info import get_all_analysis_ply_names
+from ansys.dpf.composites.layup_info.material_properties import MaterialProperty
+from ansys.dpf.composites.ply_wise_data import SpotReductionStrategy, get_ply_wise_data
+from ansys.dpf.composites.select_indices import get_selected_indices
+from ansys.dpf.composites.server_helpers import connect_to_or_start_server
+
+# %%
+# Start a DPF server and copy the example files into the current working directory.
+server = connect_to_or_start_server()
+composite_files = get_continuous_fiber_example_files(server, "cyclic_symmetry")
+
+# %%
+# Create a composite model
+composite_model = CompositeModel(composite_files, server)
+
+# %%
+# Evaluate a combined failure criterion
+combined_failure_criterion = CombinedFailureCriterion(failure_criteria=[MaxStressCriterion()])
+failure_result = composite_model.evaluate_failure_criteria(combined_failure_criterion)
+
+# %%
+# Plot the failure results
+irf_field = failure_result.get_field({"failure_label": FailureOutput.FAILURE_VALUE})
+irf_field.plot()
+
+# %%
+# Plot ply-wise stresses
+# ~~~~~~~~~~~~~~~~~~~~~~
+# All functions in PyDPF Composites can be used to
+# post-process the original (first) sector.
+
+rst_stream = composite_model.core_model.metadata.streams_provider
+stress_operator = dpf.operators.result.stress()
+stress_operator.inputs.streams_container.connect(rst_stream)
+stress_operator.inputs.bool_rotate_to_global(False)
+stress_container = stress_operator.outputs.fields_container()
+
+all_ply_names = get_all_analysis_ply_names(composite_model.get_mesh())
+all_ply_names
+
+component_s11 = Sym3x3TensorComponent.TENSOR11
+stress_field = stress_container[0]
+elemental_values = get_ply_wise_data(
+    field=stress_field,
+    ply_name="P3L1__ModelingPly.1",
+    mesh=composite_model.get_mesh(),
+    component=component_s11,
+    spot_reduction_strategy=SpotReductionStrategy.MAX,
+    requested_location=dpf.locations.elemental,
+)
+composite_model.get_mesh().plot(elemental_values)
+
+# %%
+# Custom failure criterion
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# The following code block shows how to implement a custom failure criterion.
+# It computes the inverse reserve factor for each element with respect to
+# fiber failure. The criterion distinguishes between tension and compression.
+
+# Prepare dict with the material properties
```

Comment:
```suggestion
# Prepare a dictionary (dict) with the material properties.
```

Reply:
@roosre Is elaborating on dict benefical? Or do the users know what it means already?

Reply:
Hi @nshum4 , a python user knows what a dict is. It's a native python class (object) and widely used. Thanks, René

Reply:
```suggestion
# Prepare dict with the material properties.
```